src/boards/include/boards/pico_w.h

```diff
@@ -99,4 +99,22 @@
 #define CYW43_WL_GPIO_LED_PIN 0
 #endif
 
+// CYW43 GPIO to get VBUS
+#ifndef CYW43_WL_GPIO_VBUS_PIN
+#define CYW43_WL_GPIO_VBUS_PIN 2
```

Comment:
Slight inconsistency in the order-of-words in `PICO_VBUS_GPIO_PIN` and `CYW43_WL_GPIO_VBUS_PIN` but I dunno if that's deliberate and/or if it matters? :wink:

Reply:
I've dropped "GPIO" from the pin defs (except for the WL one). I think that's consistent?